libpod/runtime.go

```diff
@@ -462,17 +462,20 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 		}
 	}
 
-	// If we need to make a default network - do so now.
-	if err := defaultnet.Create(runtime.config.Network.DefaultNetwork, runtime.config.Network.DefaultSubnet, runtime.config.Network.NetworkConfigDir, runtime.config.Engine.StaticDir, runtime.config.Engine.MachineEnabled); err != nil {
```

Comment:
This is done by the ContainerNetwork as it initializes now?

Reply:
The default network is now in memory only. We won't write a cni config file. That allows the user to change the default network name/subnet in containers.conf and the changes will be picked up by the next podman command.